### PR TITLE
Cleanup controllers

### PR DESCRIPTION
**Pull request by vincepri**
Signed-off-by: Vince Prignano <vincepri@vmware.com>

**What this PR does / why we need it**:
This PR is an effort to cleanup comments, codebase, logs and some inconsistencies found in the controllers code. 

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

_Please confirm that if this PR changes any image versions, then that's the sole change this PR makes._

**Release note**:

```release-note

```

### CODE REVIEW

**Comment by davidewatson** [comment]
I'll leave this until you remove the `WIP`...
/retest
/hold

File: pkg/controller/machinedeployment/sync.go [hunk 1]
```diff
@@ -492,17 +492,16 @@ func updateMachineDeployment(c client.Client, d *clusterv1alpha1.MachineDeployme
...
 			return err
 		}
-
+		// Apply defaults.
 		clusterv1alpha1.PopulateDefaultsMachineDeployment(d)
-		// Apply modifications
+		// Apply modifications.
 		modify(d)
-		// Update the machineDeployment
+		// Update the achineDeployment.
```
Comment (tahsinrahman) [inline pkg/controller/machinedeployment/sync.go]: achineDeployment -> machineDeployment

**Pull request merged**